backend/src/audit_log/audit_event.rs

```diff
@@ -0,0 +1,207 @@
+use axum::http::Uri;
+use chrono::{DateTime, NaiveDate, Utc};
+use serde::{self, Deserialize, Serialize};
+use strum::VariantNames;
+use utoipa::ToSchema;
+
+use crate::{
+    ErrorResponse,
+    authentication::LoginResponse,
+    data_entry::PollingStationDataEntry,
+    election::{Election, ElectionCategory, ElectionStatus},
+    error::ErrorReference,
+    polling_station::{PollingStation, PollingStationType},
+};
```

Comment:
Can we find a way to create these details structs without a reference to each module? Each of these module already has a reference to `audit_log`, so we now have cyclic dependencies.

Reply:
Yeah that is not great, but we either way need the types, like `ElectionStatus` or `PollingStationType`. Alternatively I could not use these types but store them as a string. Than I could move al the From<T> stuff to an Into<T>, which makes the cyclic dependencies go away. What do you prefer @praseodym?

Reply:
Hmm, we would also get a lot of clippy warnings, since `From` is preferred over `Into`

Reply:
What about making all types in the `*Details` structs a type from `std` (e.g. make `election_category` and `election_status` in `ElectionDetails` a `String`) and then moving the `From` trait implementation to the module (e.g. `impl From<Election> for ElectionDetails` to the `election` module)? That should remove the references from the `audit_log` module to every other module.

Reply:
So exactly what I did in https://github.com/kiesraad/abacus/commit/ebb5a162d13ad77b31463e090566e344e15b2c37? Then I will push that commit to this branch.

Reply:
Yes that looks good!